server/templates.go

```diff
@@ -15,23 +15,23 @@ func ValidTemplateRequest(template *chronograf.Template) error {
 	switch template.Type {
 	default:
 		return fmt.Errorf("Unknown template type %s", template.Type)
-	case "query", "constant", "csv", "fieldKeys", "tagKeys", "tagValues", "measurements", "databases", "map":
+	case "query", "constant", "csv", "fieldKeys", "tagKeys", "tagValues", "measurements", "databases", "map", "influxql":
```

Comment:
if `query` is getting replaced with `influxql`, should `query` still be a valid option for template type?

Reply:
Ah whoops. This snuck in again after resolving a merge conflict.

👍